openapi3/schema_test.go

```diff
@@ -17,12 +18,11 @@ type schemaExample struct {
 	Title         string
 	Schema        *Schema
 	Serialization interface{}
-	AllValid      []interface{}
-	AllInvalid    []interface{}
+	AllValid      [][]interface{} // indexed by minor OpenAPI version supported
+	AllInvalid    [][]interface{} // also indexed by minor OpenAPI version supported
```

Comment:
I appreciate your work and these are quite a lot of changes but please try to keep changes to this file to a minimum and extract if possible the tests that rely on minor version to another part of this file or to another file dedicated to said changes. This file is quite dense and review is difficult already. If that's just too much work let's concentrate on the other comments. Thanks!

Reply:
I haven't gone through this one yet. Will keep you posted.